src/import/instruments.ts

```diff
@@ -361,7 +361,11 @@ async function readFormTemplate(tree: TraceDirectoryTree): Promise<FormTemplateD
   const archive = readInstrumentsKeyedArchive(await readAsArrayBuffer(formTemplate))
 
   const version = archive['com.apple.xray.owner.template.version']
-  const selectedRunNumber = archive['com.apple.xray.owner.template'].get('_selectedRunNumber')
+  let selectedRunNumber = 1
+  if ('com.apple.xray.owner.template' in Object.keys(archive)) {
```

Comment:
I don't think this is quite right. The `in` test is for objects, not arrays, and `Object.keys` returns an array, so I think this condition will always be false.

![image](https://user-images.githubusercontent.com/150329/52146341-4fb76380-2618-11e9-9d31-e09606e68a6f.png)


Instead, I think you want:

```suggestion
  if ('com.apple.xray.owner.template' in archive) {
```